app/models/user.rb

```diff
@@ -38,10 +57,33 @@ class User < ApplicationRecord
            :organization,
            :password_digest, to: :phone_number_authentication, allow_nil: true
 
+  delegate :email,
+           :password,
+           :authenticatable_salt,
+           :invited_to_sign_up?, to: :email_authentication, allow_nil: true
+
+  def self.invite!(options = {})
+    transaction do
+      now = Time.now
+      User.create(
+        options
+          .slice(:full_name, :role)
+          .merge(device_created_at: now,
+                 device_updated_at: now,
+                 sync_approval_status: sync_approval_statuses[:denied]))
```

Comment:
Should the default sync approval status be denied?

Reply:
Yes, the admins don't require syncing.